pkg/loadbalancer/loadbalancer.go

```diff
@@ -59,6 +60,7 @@ const (
 	serviceFlagLoadBalancer    = 32
 	serviceFlagRoutable        = 64
 	serviceFlagSourceRange     = 128
+	serviceFlagLocalRedirect   = 256
```

Comment:
nit: Per the above comment, it would be more obvious to pick up on the discrepancy if these were defined using bit definitions like they are on the datapath side rather than the decimal integer values. (Could be a follow-up)

Reply:
Will fix it. Also, looks like there is a unit test for this code. I'll also update the test case with the new flag that I added.